contact.html

```diff
@@ -1,137 +1,7 @@
-                                                <button type="submit" class="theme-btn btn-style-three">Send</button>
-                                                <span id="contact-form-submit-loader" class="submit-loader" />
-                                            </div>
-                                        </div>
-                                        <div id="contact-form-feedback" class="form-group form-feedback display-none">Thanks for your feedback.</div>
-                                    </form>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </section>
-            <footer class="main-footer">
-                <div class="footer-bottom">
-                    <div class="auto-container">
-                        <div class="row clearfix">
-                            <div class="col-md-6 col-sm-12 col-xs-12">
-                                <div class="copyright">&copy; 2018 by <a href="//clapware.com/">Clapware</a></div>
-                            </div>
-                            <div class="col-md-6 col-sm-12 col-xs-12">
-                                <div class="footer-nav clearfix">
-                                    <ul class="clearfix">
-                                        <li><a href="termsofservice.html">Terms of Service</a></li>
-                                        <li><a href="privacypolicy.html">Privacy Policy</a></li>
-                                        <li><a href="helpcenter.html">Help Center</a></li>
-                                        <li><a href="contact.html">Contact</a></li>
-                                    </ul>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </footer>
-        </div>
-        <div class="scroll-to-top scroll-to-target" data-target=".main-header"><span class="fa fa-long-arrow-up" /></div>
-        <link href="css/style.min.css" rel="stylesheet">
-        <link href="css/responsive.min.css" rel="stylesheet">
-        <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
-        <script src="//cdnjs.cloudflare.com/ajax/libs/wow/1.1.2/wow.min.js"></script>
-        <script src="//cdnjs.cloudflare.com/ajax/libs/jquery-validate/1.17.0/jquery.validate.min.js"></script>
-        <script src="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" async></script>
-        <script src="//cdnjs.cloudflare.com/ajax/libs/animejs/1.0.0/anime.min.js" async></script>
-        <script src="//cdnjs.cloudflare.com/ajax/libs/jquery-one-page-nav/3.0.0/jquery.nav.min.js"></script>
-        <script src="js/contact.min.js" async></script>
-        <script src="js/script.min.js" async></script>
-    </body>
+    <p>Redirecting to <b>youclap.tech</b>... ⏳</p>
```

Comment:
I think that you should put "youclap.tech" as a link.
Only for case of an error occurs, after the page load and before the redirect.

Reply:
i will open a new PR with that.